lib/improver/wind_downscaling.py

```diff
@@ -641,10 +645,17 @@ def __init__(self, a_over_s_cube, sigma_cube, pporo_cube,
             any RC
 
         """
-        a_over_s_cube.data = a_over_s_cube.data.astype(np.float32)
-        sigma_cube.data = sigma_cube.data.astype(np.float32)
-        pporo_cube.data = pporo_cube.data.astype(np.float32)
-        modoro_cube.data = modoro_cube.data.astype(np.float32)
+        enforce_float32_precision([a_over_s_cube,
+                                   sigma_cube,
+                                   pporo_cube,
+                                   modoro_cube,
+                                   z0_cube,
+                                   height_levels_cube])
+
+        # Standard Python 'float' type is either single or double depending on
+        # system and no reliable method of finding which from variable. So
```

Comment:
Do you mean: `Standard Python 'float' type is either single or double depending on system and there is no reliable method of finding which from variable`?